### DISCUSSION TITLE
Linkerd API controlling the traffic to external service?

### DISCUSSION THREAD

**Discussion by qts0312** [linkerd/linkerd2#12224] [Q&A] [answered]
In my cluster, I want to control the traffic targeting at `www.google.com` and redirect to specified internal Pods. In Istio, I can use ServiceEntry to implement this function. So that, I wonder if Linkerd has API implementing similar function to control the traffic to external service. 

Thanks for your kind help.

**Comment by wmorgan** [comment, marked as answer]
Hi @qts0312, today Linkerd supports this kind of thing for internal names via TrafficSpits, but not for external destinations like you are describing. However, in the next version we are working on improved egress control capabilities and we'll probably capture this use case then.

**Reply by qts0312** [reply]
I see. Thank you.
